filament/src/driver/opengl/ContextManagerWebGL.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright (C) 2018 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "driver/opengl/ContextManagerWebGL.h"
+#include "driver/opengl/OpenGLDriver.h"
+
+namespace filament {
+
+using namespace driver;
+
+std::unique_ptr<Driver> ContextManagerWebGL::createDriver(void* const sharedGLContext) noexcept {
+    return OpenGLDriver::create(this, sharedGLContext);
+}
+
+void ContextManagerWebGL::terminate() noexcept {
+}
+
+ExternalContext::SwapChain* ContextManagerWebGL::createSwapChain(
+        void* nativeWindow, uint64_t& flags) noexcept {
+    return (SwapChain*) nativeWindow;
+}
+
+void ContextManagerWebGL::destroySwapChain(ExternalContext::SwapChain* /*swapChain*/) noexcept {
```

Comment:
Any reason `swapChain` is in comments here but not elsewhere?

Reply:
I'll make this consistent.